conf/app_server.yaml

```diff
@@ -24,6 +24,8 @@ email_password: 19782013
 email_from: info@bh.org.il
 
 # Bucket urls
+bagnowka_bucket_url: https://s3-us-west-2.amazonaws.com/bagnowka-scraped/full
+bagnowka_bucket: full
```

Comment:
1. The bagnowka_bucket configuration is there for future use, when more buckets will be added (like bialystock). I figured it's best to not upload all scraped images (from external sources) to the same bucket, in case we will need to transfer them or whatever. However, since for now it's not in use, I'll remove it.
2. changes in files like .noseids / bhdata.tgz always occur when updating my local branch, how should I avoid this?